libraries/consul_installation_binary.rb

```diff
@@ -86,6 +86,7 @@ def self.binary_basename(node, resource)
         case node['kernel']['machine']
         when 'x86_64', 'amd64' then ['consul', resource.version, node['os'], 'amd64'].join('_')
         when /i\d86/ then ['consul', resource.version, node['os'], '386'].join('_')
+        when /^arm/ then ['consul', resource.version, node['os'], 'arm'].join('_')
```

Comment:
What's the exact value of `node['kernel']['machine']` in your case? Can we just match it with a string `arm`, without regex?